tests/libres_tests/res/enkf/test_get_active_index_list.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from res.enkf.active_list import ActiveList
+from res.enkf.enums.active_mode_enum import ActiveMode
+
+
+def test_active_index_list():
```

Comment:
Feel free to move this into the existing file. Also, this should be split into multiple tests testing the different aspects individually.

Reply:
See my comments above. Fixed.